.commonfiles.sha

```diff
@@ -1 +1 @@
-950d08195523c9f44f15e7283274259ee0ace593
+dd59c4efb573c9046fb80cbbef1c28382f58e8b6
```

Comment:
What is this?

Reply:
The sha of the istio/common-files repo when the copy of the common-files was inserted into this repo.